Bottom: a1b7cabb10798e0b1ae089debb58afb69091bf52
Top:    99b70675ea68c5f72ba03a31cab6e6df26ce501d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-28 03:55:02 +0000

Refresh of odds-and-sods

---

diff --git a/src/polygon.h b/src/polygon.h
index aabc096..1343d03 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 10 //40
+#define POLY_CIRC_SEGS 8 //10 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
